docs/team/github_jaesimin.adoc

```diff
@@ -0,0 +1,95 @@
+== Project Portfolio for StudyBuddy Pro
+:site-section: AboutUs
+:imagesDir: ../images
+:stylesDir: ../stylesheets
+
+=== Done By: Jasmine Yeo Jia Min
+---
+
+This portfolio is meant to be a well-documented data point of my Software Engineering (SE) experience with regards to the project. It showcases my contributions to the project and my documentation skills.
+
+---
+
+=== Overview
+
+StudyBuddy Pro is a desktop Command Line Interface (CLI) application intended for Computing School students of National University of Singapore (NUS). It aims to lessen target users' revision time consumption without compromising the quality of studies with a centralized platform.
+
+StudyBuddy Pro provides three main features: Flashcard, Notes and Cheatsheet. These features are used to revise using flashcards, take notes and generate cheatsheets respectively. Additionally, StudyBuddy Pro is equipped with a Graphic User Interface (GUI).
+
+=== Syntax
+
+The below syntax notations are used throughout the document. Hence, it is good to take note of them.
+
+- `syntax 1`
+
+    For commands or class objects
+
+- _syntax 2_
+
+    For figures, tables, functionalities or variables
+
+=== Summary of contributions
+
+The sections below are a summary of my contributions to the project. It highlights my major contributions and states my minor, code and other contributions.
+
+==== Major Enhancement 1: Automated the Generation of Cheatsheets
+
+This enhancement allows users to automatically generate cheatsheets according to the _tags_ that they specify. From this, it improves the product significantly as users can sift out information to be included in a specific cheatsheet. Thus, it reduces the users' time taken in collating cheatsheets.
+
+This enhancement's implementation makes use of the other features in the StudyBuddy Pro to create the contents in the cheatsheets. As such, users can make full use of this auto-generation enhancement by using _tagging_ functionality in those features.
+
+==== Major Enhancement 2: Customization of Generated Cheatsheets
+
+This enhancement allows users to remove contents in the generated cheatsheets by specifying its content indexes or tags. From this, it improves the cheatsheets' usability as users can determine which contents to be included or excluded in the cheatsheets.
+
+This enhancement ensures that all contents included are categorized under the _tags_ specified even after users' personalization. Hence, users can be assured that no irrelevant contents will appear in the cheatsheets.
+
+==== Minor Enhancement 1: Switching between the Features
+
+This enhancement allows users to toggle amongst the features in the StudyBuddy Pro. Also, it allows _global_ commands, like `exit` or `filterall` to be used in any features.
+
+==== Minor Enhancement 2: Clearing Specific Features's Data
+
+This enhancement allows users to clear specific feature's data in the StudyBuddy Pro. As such, users do not need to clear the entire data if desired.
```

Comment:
Might want to change to StudyBuddyPro to ensure consistency with our DG/UG as well as for the rest of the occurrences in the document :)

Reply:
I thought it is "StudyBuddy Pro". I shall change it now.

Reply:
I thought so too and that probably makes more sense, shall we discuss what to finalize the name as during our next meeting? For now shall we stick to StudyBuddyPro since we used that in the DG/UG and change one-shot if we decide to?

Reply:
Sure! See you tmr :)